frontend/components/markup-modal/builder-common.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { StyledInput } from '../common'
+
+const BuilderOuterContainer = styled.div`
+  margin-top: 10px;
+  margin-bottom: 10px;
+`
+
+// The inner container is inline-block so that its width matches its columns.
+const BuilderInnerContainer = styled.div`
+  display: inline-block;
+
+  padding: 11px 14px 10px;
+
+  border-radius: 4px;
+  background: #eef;
+`
+
+const BuilderContainer = ({ children }) => (
+  <BuilderOuterContainer>
+    <BuilderInnerContainer>{children}</BuilderInnerContainer>
+  </BuilderOuterContainer>
+)
+BuilderContainer.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]),
+}
+
+const BuilderLabel = styled.label`
+  font-family: system-ui;
+  font-size: 11px;
+  text-transform: lowercase;
+`
+
+const BuilderInput = StyledInput
+
+const BuilderCaption = styled.span`
+  color: #999;
+
+  font-family: system-ui;
+  font-size: 11px;
+  text-transform: lowercase;
```

Comment:
There probably is but I couldn't figure it out. [There's a way to extend styling](https://www.styled-components.com/docs/basics#extending-styles) which we're using all over here, but it doesn't work here because they're two different elements.

I sent out a couple requests for feedback linking here; hopefully some folks will weigh in!